testsuite/libmapiproxy/openchangedb_logger.c

```diff
@@ -514,6 +539,24 @@ static const char **get_folders_names(TALLOC_CTX *mem_ctx, struct openchangedb_c
 }
 // ^ openchangedb -------------------------------------------------------------
 
+// v replica mapping
+
+static enum MAPISTATUS replica_mapping_guid_to_replid(struct openchangedb_context *self, const char *username, const struct GUID *guid, uint16_t *replid_p)
+{
+        functions_called.guid_to_replid++;
+        *replid_p = 0x03;
+        return MAPI_E_SUCCESS;
+}
+
+static enum MAPISTATUS replica_mapping_replid_to_guid(struct openchangedb_context *self, const char *username, uint16_t replid, struct GUID *guid)
+{
+        functions_called.replid_to_guid++;
+        guid = talloc_zero(self, struct GUID);
```

Comment:
Aren't you already allocating it in calling function?
https://github.com/sixstone-qq/openchange/blob/ejhernandez/distributed-replica-mapping/testsuite/libmapiproxy/openchangedb_logger.c#L205